packages/retail-ui-extensions/src/components/List/List.ts

```diff
@@ -27,6 +27,11 @@ export interface ListRowRightSide {
    * The main label that will be displayed on the right side of the row.
    */
   label?: string;
+  /**
+   * Show a chevron. Set this to true if pressing on the row navigates to another screen.
+   * @defaultValue `false`
+   */
+  showDisclosureIndicator?: boolean;
```

Comment:
Maybe something like this is better? What's a better name for this? 

```suggestion
  chevron?: boolean;
```